src/app/app.component.scss

```diff
@@ -39,10 +39,6 @@ body {
   margin: 0;
 }
 
-ds-header-navbar-wrapper {
```

Comment:
Why remove the header navbar's z-index?
If it needs to change I'd prefer you change the variable value instead of removing it entirely